proposed/phpdoc.md

```diff
@@ -1220,18 +1220,19 @@ The @method allows a class to know which 'magic' methods are callable.
 
 #### Syntax
 
-    @method [return type] [name]([type] [parameter], [...]) [description]
+    @method <static|> [name]([type] [parameter], [...])[: return type] [description]
```

Comment:
@teohhanhui Dunno, if I understand you correctly, but this section _uses_ ABNF.

Thus, either `name` or `<name>` should be used IMO; and to avoid ambiguity, I'm in favor of `<name>`.